src/Copy/Text.elm

```diff
@@ -135,36 +135,36 @@ t key =
         NotAloneTitle ->
             "You are not alone"
 
-        -- \u{00A0} is a non-breaking space that prevents a single word wrapping to a new line
         Journey1Teaser ->
-            "He threw my phone on the floor and smashed\u{00A0}it"
+            "He considered my money\u{00A0}his"
 
         Journey1Image ->
-            "/Person_1.svg"
+            "/Person_2.svg"
```

Comment:
@Nikomus Do you think we want to swap the women or the numbers?

Reply:
I was going to leave, we don't particularly correspond the illustrations to the stories, I don't think.